netty/src/test/java/io/grpc/netty/WriteQueueTest.java

```diff
@@ -61,14 +62,16 @@
 @RunWith(JUnit4.class)
 public class WriteQueueTest {
 
+  private final Object lock = new Object();
+
   @Mock
   public Channel channel;
 
   @Mock
   public ChannelPromise promise;
 
-  private long flushCalledNanos;
   private long writeCalledNanos;
+  private long flushCalledNanos = writeCalledNanos + 1;
```

Comment:
This appears to have been correct previously, since flushing at least once is required.